src/components/Search/SearchOverTime.tsx

```diff
@@ -138,7 +138,7 @@ export default function SearchOverTime({
           <Grid
             item
             xs
-            style={{ maxWidth: '360px' }}
+            style={{ maxWidth: '364px' }}
```

Comment:
do you know why it's 1px smaller than below?

Reply:
This is a typo. Thanks for catching it.